### ISSUE TITLE
Two wrong URLs

### ISSUE THREAD

**Issue by massic80** [w3c/wai-wcag-quickref#164]
In https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html there are two wrong hashes:

- https://www.w3.org/TR/WCAG21/#dfn-user-interface-component should be https://www.w3.org/TR/WCAG21/#dfn-user-interface-components
- https://www.w3.org/TR/WCAG21/#dfn-changes-of-context should be https://www.w3.org/TR/WCAG21/#dfn-change-of-context

**Comment by yatil** [comment]
Hi @massic80, I will move this issue to the repository (https://github.com/w3c/wcag) where it can be addressed :-)

**Comment by yatil** [comment]
This issue was moved to w3c/wcag#338
